scripts/testEnv.sh

```diff
@@ -129,9 +129,24 @@ export ORION_DATABASE_PASSWORD=${ORION_DATABASE_PASSWORD:-orion}
 # o CP4_DATABASE_NAME      - database for fourth context provider
 # o CP5_DATABASE_NAME      - database for fifth context provider
 #
-export CB_DATABASE_NAME=${CB_DATABASE_NAME:-ftest}
-export CP1_DATABASE_NAME=${CP1_DATABASE_NAME:-ftest_cp1}
-export CP2_DATABASE_NAME=${CP2_DATABASE_NAME:-ftest_cp2}
-export CP3_DATABASE_NAME=${CP3_DATABASE_NAME:-ftest_cp3}
-export CP4_DATABASE_NAME=${CP4_DATABASE_NAME:-ftest_cp4}
-export CP5_DATABASE_NAME=${CP5_DATABASE_NAME:-ftest_cp5}
+
+
+export DATABASE_NAME_BASE=CB
```

Comment:
Fixed in 3ef4ea3 and bd93c95, the name of the variable has been changed from CB_DATABASE_NAME to CB_DB_NAME .If it is better to change to FTEST_DB_NAME tell me and I will change it.It has been changed for all the test that uses that variable.